win-odbc64/Install/win64_installer/installer.iss

```diff
@@ -152,6 +141,40 @@ begin
   Result := (GetUninstallString() <> '');
 end;
 
+procedure CurStepChanged(CurStep: TSetupStep);
+var
+  bRedistNeedsInstall: Boolean;
+  sDefaultBrowserName: String;
+  sDefaultBrowserReg: String;
+  sDefaultBrowserValue: String;
+  iQuotePos: Integer;
+  lLength: Longint;
+  iRetCode: Integer;
+begin
+  if CurStep=ssPostInstall then
+  begin
+    bRedistNeedsInstall := VCRedistNeedsInstall();
+    if bRedistNeedsInstall=True then
+    begin
+      MsgBox('Install of Visual C++ Redistributable Packages for Visual Studio 2013 is not dectected.'+ #13#10#13#10 + 'Please download vcredist_x64.exe from Microsoft Download Centre and install it.', mbInformation, MB_OK);
```

Comment:
Can we change the message to be as follows:
Run-time dependency needed by the driver is not found. Proceeding to download and install run-time dependency from Microsoft.